tasks/gitlab_helpers.py

```diff
@@ -33,11 +31,12 @@
 
 @task
 def generate_ci_visibility_links(_ctx, output: str | None):
+    """Generates links to CI Visibility for the current job.
+
+    Usage:
```

Comment:
Could  we make the usage the same between the tasks. Either use `deva` or `inv` everywhere but not a mix of the two?